.github/workflows/markdown-style-checks.yml

```
@@ -19,6 +19,10 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
-    - uses: DavidAnson/markdownlint-cli2-action@v16
+    - uses: DavidAnson/markdownlint-cli2-action@v19
       with:
         config: "docs/.sphinx/.markdownlint.json"
+        globs: |
+          README.md
+          CHANGELOG.md
+          docs/*.md
```

```suggestion
          docs/**.{md,markdown}
```

As per the [doc](https://github.com/DavidAnson/markdownlint-cli2#use), '`* matches any number of characters, but not /`'. Files in sub-folders (e.g. `docs/tutorials/foo.md`) won't match.
However `** matches any number of characters, including /` and is thus better suited here.